app/poros/quote.rb

```diff
@@ -0,0 +1,10 @@
+class Quote
+  attr_reader :id, :content, :author, :tags
+
+  def initialize(data)
+    @id = data[:_id]
+    @content = data[:content]
+    @author = data[:author]
+    @tags = data[:tags].to_sentence
+  end
+end
```

Comment:
I like that you're adding your id in here. It can definitely help with testing later on.